documentation/source/user_section/tutorials/spinalcord-segmentation/sct_propseg-example-t2.rst

```diff
@@ -0,0 +1,58 @@
+Hands-on Example: T2
+####################
+
+Run the following command to process the image:
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_qc.png
+  :align: right
+  :figwidth: 40%
+
+  The QC report for the segmented image
+
+.. code:: sh
+
+   sct_propseg -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
```

Comment:
for some weird reason, the QC report is not created 🤔

<details><summary>Terminal output</summary>

```console
julien-macbook:~/Desktop/single_subject/data/t2 $ sct_propseg -i t2.nii.gz -c t2 -qc ~/qc_singleSubj 
--
Spinal Cord Toolbox (git-master-c951389c917884e7e3bb3c3f39aea47669a07a79)

sct_propseg -i t2.nii.gz -c t2 -qc /Users/julien/qc_singleSubj
--

Creating temporary folder (/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211356.916429-label_vertebrae-c0sb7urp)
Creating temporary folder (/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211357.015759-propseg-d8dtd2_a)
Creating temporary folder (/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211357.276402-y9wm7wpb)
Remove temporary files...
rm -rf /var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211357.276402-y9wm7wpb
/Users/julien/code/sct/bin/isct_propseg -t t2 -o . -verbose -mesh -centerline-binary -CSF -centerline-coord -cross -init-tube -low-resolution-mesh -i /var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211357.015759-propseg-d8dtd2_a/t2_rescaled.nii.gz -init-centerline /var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211356.916429-label_vertebrae-c0sb7urp/centerline_optic.nii.gz -centerline-binary # in /Users/julien/Desktop/single_subject/data/t2
mv ./t2_rescaled_seg.nii.gz ./t2_seg.nii.gz
mv ./t2_rescaled_centerline.nii.gz ./t2_centerline.nii.gz

Check consistency of segmentation...
Creating temporary folder (/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211428.679489-propseg-45vdbcgx)
/private/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211428.679489-propseg-45vdbcgx/tmp.segmentation.nii.gz
/private/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211428.679489-propseg-45vdbcgx/tmp.centerline.nii.gz

Get data dimensions...
/private/var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211428.679489-propseg-45vdbcgx/tmp.segmentation_RPI_c.nii.gz
rm -rf /var/folders/s8/4qnm5q1n261ch35b5kkclsb00000gn/T/sct-20210603211428.679489-propseg-45vdbcgx
Copy header input --> output(s) to make sure qform is the same.
File /Users/julien/Desktop/single_subject/data/t2/t2_seg.nii.gz already exists. Will overwrite it.
File /Users/julien/Desktop/single_subject/data/t2/t2_centerline.nii.gz already exists. Will overwrite it.
/Users/julien/code/sct/python/envs/venv_sct/lib/python3.6/site-packages/scipy/ndimage/measurements.py:1391: RuntimeWarning: invalid value encountered in double_scalars
  for dir in range(input.ndim)]
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/js/filesaver.min.js /Users/julien/qc_singleSubj/_assets/js
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/js/yaml.min.js /Users/julien/qc_singleSubj/_assets/js
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/imgs/f-icon.png /Users/julien/qc_singleSubj/_assets/imgs
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/imgs/.DS_Store /Users/julien/qc_singleSubj/_assets/imgs
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/imgs/keyright.png /Users/julien/qc_singleSubj/_assets/imgs
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/imgs/keydown.png /Users/julien/qc_singleSubj/_assets/imgs
cp /Users/julien/code/sct/spinalcordtoolbox/reports/assets/_assets/imgs/keyup.png /Users/julien/qc_singleSubj/_assets/imgs

Done! To view results, type:
fsleyes /Users/julien/Desktop/single_subject/data/t2/t2.nii.gz -cm greyscale /Users/julien/Desktop/single_subject/data/t2/t2_seg.nii.gz -cm red -a 100.0 &

julien-macbook:~/Desktop/single_subject/data/t2 $ ls -la
total 21880
drwxr-xr-x@ 18 julien  staff      576  3 Jun 21:14 .
drwxr-xr-x@  3 julien  staff       96  3 Jun 21:04 ..
-rw-r--r--   1 julien  staff     6812  3 Jun 21:14 InitialTube1.vtk
-rw-r--r--   1 julien  staff     6736  3 Jun 21:14 InitialTube2.vtk
-rw-r--r--   1 julien  staff     6790  3 Jun 21:14 InitialTubeCSF1.vtk
-rw-r--r--   1 julien  staff     6731  3 Jun 21:14 InitialTubeCSF2.vtk
-rw-r--r--   1 julien  staff   199439  3 Jun 21:14 segmentation_CSF_mesh_low_resolution.vtk
-rw-r--r--   1 julien  staff   140729  3 Jun 21:14 segmentation_mesh_low_resolution.vtk
-rw-r--r--@  1 julien  staff  9092309  2 Jun 19:59 t2.nii.gz
-rw-r--r--   1 julien  staff    30065  3 Jun 21:14 t2_centerline.nii.gz
-rw-r--r--@  1 julien  staff   229009  2 Jun 20:05 t2_labels_vert.nii.gz
-rw-r--r--   1 julien  staff   807878  3 Jun 21:14 t2_rescaled_CSF_mesh.vtk
-rw-r--r--   1 julien  staff    19994  3 Jun 21:14 t2_rescaled_CSF_seg.nii.gz
-rw-r--r--   1 julien  staff    10904  3 Jun 21:14 t2_rescaled_centerline.txt
-rw-r--r--   1 julien  staff     2750  3 Jun 21:14 t2_rescaled_cross_sectional_areas.txt
-rw-r--r--   1 julien  staff     3665  3 Jun 21:14 t2_rescaled_cross_sectional_areas_CSF.txt
-rw-r--r--   1 julien  staff   576261  3 Jun 21:14 t2_rescaled_mesh.vtk
-rw-r--r--   1 julien  staff    37631  3 Jun 21:14 t2_seg.nii.gz
```

</details>

Reply:
ah! my bad. It is created under `~/`.

However, the weird thing is that there is no more the message: `QC created, open with the following command: open blablabla`. Maybe it is a consequence of a recent commit?

Reply:
hum... the QC report message did appear in the Terminal output with `sct_register_to_template` but not with `sct_warp_template`. And I noticed this RuntimeWarning:
```console
te/PAM50_label_spinal_levels.nii.gz -o label/template/PAM50_label_spinal_levels.nii.gz -t warp_template2anat.nii.gz -r t2.nii.gz -n NearestNeighbor # in /Users/julien/Desktop/single_subject/data/t2
cp /Users/julien/code/sct/data/PAM50/template/info_label.txt label/template
/Users/julien/code/sct/python/envs/venv_sct/lib/python3.6/site-packages/scipy/ndimage/measurements.py:1391: RuntimeWarning: invalid value encountered in double_scalars
  for dir in range(input.ndim)]

Done! To view results, type:
fsleyes t2.nii.gz -cm greyscale -a 100.0 label/template/PAM50_t2.nii.gz -cm greyscale -dr 0 4000 -a 100.0 label/template/PAM50_gm.nii.gz -cm red-yellow -dr 0.4 1 -a 50.0 label/template/PAM50_wm.nii.gz -cm blue-lightblue -dr 0.4 1 -a 50.0 &
```
Could it be related?  Note that this warning did not appear with `sct_propseg`.
Also note that the QC report for `sct_warp_template` _was_ created (it was just not announced in the Terminal output).

Reply:
Ainsleigh also noticed this, and it was indeed due to a bug. I have opened a PR for this, see: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3094#discussion_r646000016 :slightly_smiling_face: